portal/tests/test_security.py

```diff
@@ -104,4 +104,4 @@ def test_class_page_wrong_teacher(self):
     def test_anonymous_cannot_access_teaching_materials(self):
         c = Client()
         page = reverse_lazy('materials')
-        self.assertNotEqual(c.get(page).status_code, 200)
+        self.assertEqual(c.get(page).status_code, 302)
```

Comment:
`302`? Where is the user redirected to? If the user is allowed to access, they get a `302` too right?

Maybe we should be checking where the user is sent to?